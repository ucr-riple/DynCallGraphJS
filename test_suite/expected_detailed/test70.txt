{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test70.js:1:1:17:3)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test70.js:1:2:8:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test70.js:7:3:7:21)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test70.js:13:14:15:6)'
    ]
}