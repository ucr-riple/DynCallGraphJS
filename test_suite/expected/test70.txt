{ '(test70.js:1:1:17:3)':
   [ '(test70.js:1:2:8:2)' ],
  '(test70.js:7:3:7:21)':
   [ '(test70.js:13:14:15:6)' ] }